libbeat/autodiscover/providers/kubernetes/pod.go

```diff
@@ -383,6 +383,7 @@ func (p *pod) containerPodEvents(flag string, pod *kubernetes.Pod, c *containerI
 	if len(namespaceAnnotations) != 0 {
 		kubemeta["namespace_annotations"] = namespaceAnnotations
 	}
+	orchestrator := genOrchestratorFields(kubemeta, "pod")
```

Comment:
Events here are emitted at the container level, should we use the container fields instead?